src/app/Routing.tsx

```diff
@@ -29,7 +29,7 @@ const Routing: React.FC = () => {
       <Route path="/minigames" element={<MinigamesPage />}></Route>
       <Route path="/404" element={<NoMatch />} />
       <Route
-        path="/profil"
+        path="/profile/student"
```

Comment:
Zastanawiam się, czy da się zrobić coś w stylu /profile/{accountType}, żeby path automatycznie przybierał nazwę student/teacher i zrobić to w jednym Route zamiast dwóch.

Reply:
yes it is possible